sandbox/example/build.gradle

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
nit: can we change the directory to `sandbox/examples` to be consistent w/ `plugins/examples`. I know it seems overly fastidious but I think it would be worthwhile to include a few examples that we might consider useful for trainings or blog posts.